site/docs/v1/tech/apis/scim/_scim-enterpriseuser-request-body.adoc

```diff
@@ -0,0 +1,72 @@
+Because SCIM schemas are completely customizable, There are no required fields for the incoming request other than the common fields that all SCIM requests must include to match the specification. The only requirement about the user data is that in the incoming request lambda, you must specify either the **email** or the **username** or both for the FusionAuth User object. Either of these values may be used to uniquely identify the User and may be used to authenticate the User. Otherwise, it is completely up to your defined lambda to match any other incoming user data to the appropriate FusionAuth User fields for your configuration.
+
+The following is an example of a typical SCIM User create request body your SCIM client might send. However, your incoming request lambda could use these values in completely different ways to populate the FusionAuth User object if you choose.
+
+This is taken from the SCIM Schema RFC describing a SCIM User schema. For the full specification you can find the RFC here:
+
+- https://datatracker.ietf.org/doc/html/rfc7643#page-19
+
+==== Request Body
```

Comment:
You kind of make this serve two purposes:

* a sample request body, from the spec
* suggested mappings to the FusionAuth user object

I'd suggest doing just the first, and then building a table that offers a typical mapping. This could be a different section of the API docs.